AsyncDisplayKit/Details/_ASDisplayView.mm

```diff
@@ -375,3 +378,126 @@ - (UIView *)preferredFocusedView
 }
 #endif
 @end
+
+
+#pragma mark - Accessibility
+
+static const char *ASDisplayNodeAssociatedNodeKey = "ASAssociatedNode";
+
+@implementation UIAccessibilityElement (_ASDisplayView)
+
+- (void)setAsyncdisplaykit_node:(ASDisplayNode *)node
+{
+  objc_setAssociatedObject(self, ASDisplayNodeAssociatedNodeKey, node, OBJC_ASSOCIATION_ASSIGN); // Weak reference to avoid cycle, since the node retains the layer.
```

Comment:
I'm worried about the possibility of bad access references / stale pointers...do we really need this?  In what case are we not dealing with a UIView?  Does this code ever get run, since UIAccessibilityElement is an optional object type for clients to use in their apps?